api/pkg/service/resource/resource.go

```diff
@@ -60,6 +60,7 @@ func (s *service) Query(ctx context.Context, p *resource.QueryPayload) (*resourc
 		Catalogs:   p.Catalogs,
 		Categories: p.Categories,
 		Tags:       p.Tags,
+		Platforms:  p.Platforms,
```

Comment:
could you also add this field for /v1 api?

Reply:
@sm43 the Platforms field is added to /v1 api.